docs/moz-addon-manager.md

```diff
@@ -21,95 +14,69 @@ Refresh the page, open the JavaScript console, and you should see a global `mozA
+
+To activate access to this API on a development site, first make sure you set the `extensions.webapi.testing` preference to `true`. If you're testing on an Android device then you can now access `mozAddonManager`.
+
+If you're testing on desktop Firefox to emulate Android (for development), you need to make sure your user agent looks like Firefox for Android. You can [save a custom device](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode#Saving_custom_devices) in [Responsive Design Mode](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode) with a [Firefox for Android user agent string](<https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox#Android_(version_41_and_above)>). Make sure it is touch enabled. This will let you see the add-on installation switch on desktop.
 
 ## Developing with a local server
 
-When you're running a server locally for development, you need to grant `mozAddonManager`
-access to your `localhost` domain in addition to setting the `extensions.webapi.testing` preference to `true`.
+When you're running a server locally for development, you need to grant `mozAddonManager` access to your `localhost` domain in addition to setting the `extensions.webapi.testing` preference to `true`.
 
 You can do this by building a version of Firefox with a custom patch.
 
-Start by
-[setting up your machine to build Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions).
-That page gives you a link to a helpful bootstrapping script. Run it like this:
+Start by [setting up your machine to build Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions). That page gives you a link to a helpful bootstrapping script. Run it like this:
 
 ```
 python bootstrap.py
 ```
 
-It will ask you what version of Firefox you want to build.
-Choose:
+It will ask you what version of Firefox you want to build. Choose:
 
 ```
 Firefox for Desktop
 ```
 
-Do not choose the artifact build because unfortunately you have to compile C++ code.
-It will prompt you to install some dependencies. Re-run it until you have
-everything installed and configured.
+Do not choose the artifact build because unfortunately you have to compile C++ code. It will prompt you to install some dependencies. Re-run it until you have everything installed and configured.
 
-After you have everything, the bootstrap script prompts you to
-check out a clone of [mozilla-central](https://hg.mozilla.org/mozilla-central/).
+After you have everything, the bootstrap script prompts you to check out a clone of [mozilla-central](https://hg.mozilla.org/mozilla-central/).
 
-Change into the source directory and apply this patch to allow a local server
-at a URL like `localhost:3000`.
+Change into the source directory and apply this patch to allow a local server at a URL like `localhost:3000`.
 
+<!-- prettier-ignore -->
```

Comment:
This doesn't seem to have any effect 😛  Or maybe you need to revert the first patch that formatted the diff code?

Reply:
That is, the diff code has still been altered in comparison to master.